src/Raven.Client/Documents/Changes/DatabaseChanges.cs

```diff
@@ -56,11 +58,26 @@ public DatabaseChanges(RequestExecutor requestExecutor, string databaseName, Act
             _task = DoWork();
         }
 
+        public static event Func<object, X509Certificate, X509Chain, SslPolicyErrors, bool> WebSocketsRemoteCertificateValidationCallback;
```

Comment:
Probably we want to support multiple of these. Check how it is done in RequestExecutor.

Reply:
I'm wondering if this is the right place. Now it is the only place in client using web-sockets, but if this will change? Maybe it would be best to keep it in RequestExecutor like the other callback?

Reply:
When this is changed after DatabaseChanges was created then we need to `ReconnectClient` probably.

Reply:
Let's reuse the `RequestExecuter` impl here. This way, a user only need to configure it once globally.
There is no sense in defining the same logic once for all calls and once for websockets.

Reply:
I don't believe we need multiple user validation callbacks. The implementation in RequestExecutor was done like that because of the server, registering its own callback at startup (and had a race with the callback used in tests which we register in TestBase.

I don't mind moving it to RE but I think each feature should have its own callback. So if we add a new feature in the future which uses websockets, it will not share the same callback as DatabaseChanges

Reply:
I do not see a good reason for having separate websocket callbacks per feature. Having separate callbacks in DatabaseChanges and some other feature would only be confusing. Let's try using the RE implementation if possible.